kits/docsense-agent/apps/.env.example

```
@@ -0,0 +1,4 @@
+﻿DOCSENSE_INTAKE="DOCSENSE_INTAKE Flow ID"
```

dotenv-linter (4.0.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)